application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -176,22 +178,40 @@ RestAction<Message> constructChatGptAttempt(ThreadChannel threadChannel,
             return useChatGptFallbackMessage(threadChannel);
         }
 
+        List<String> ids = new CopyOnWriteArrayList<>();
         RestAction<Message> message =
                 mentionGuildSlashCommand(threadChannel.getGuild(), ChatGptCommand.COMMAND_NAME)
                     .map("""
                             Here is an AI assisted attempt to answer your question 🤖. Maybe it helps! \
                             In any case, a human is on the way 👍. To continue talking to the AI, you can use \
                             %s.
                             """::formatted)
-                    .flatMap(threadChannel::sendMessage);
+                    .flatMap(threadChannel::sendMessage)
+                    .onSuccess(m -> ids.add(m.getId()));
+        String[] answers = chatGPTAnswer.get();
```

Comment:
`get()` is deprecated in favor of `orElseThrow()`

Reply:
not my code, but will fix it

Reply:
why though if there is a `isEmpty` check before

Reply:
`Get` does precisely the same, only difference is that `orElseThrow` has an objectively better naming.

If you read the documentation, it also states that `orElseThrow` should be used instead.

Reply:
>  only difference is that `orElseThrow` has an objectively better naming.
why though lol, it will never throw so this name is kinda misleading imo

Reply:
its not misleading. it indicates that u consider the fact that it cant be empty at this stage a critical bug. so its precisely what u want to indicate exactly that.

Reply:
like, consider it a `getOrCrash()`. meaning: if its not there, theres a bad bug in the code. in other terms: hey reader, i expect this to be filled at this point.